RFS/src/main/java/org/opensearch/migrations/bulkload/common/OpenSearchClient.java

```diff
@@ -67,7 +67,7 @@ protected OpenSearchClient(RestClient client, FailedRequestsLogger failedRequest
     }
 
     public Version getClusterVersion() {
-        return client.getAsync("/", null)
+        return client.getAsync("", null)
```

Comment:
Interesting.  So we were doing something like `curls -X GET https://<hostname>:9200//`, and the double slash was accepted for self-host versions of OpenSearch, but not the managed version?  That makes sense to me given all the processing they do.

Reply:
Yes, it was accepted on everything but 2.15 managed.